apis/match-to-id/api/riv-demo-match-to-id.yaml

```diff
@@ -0,0 +1,299 @@
+---
+openapi: 3.0.3
+
+info:
+  title: The Maryland Test Facility Match-to-ID Interface
+  description: This document specifies the API requirements for MdTF testing of algorithms that match facial biometric samples to identity document images (``match-to-id'').  Match-to-ID testing at the MdTF is supported by the Department of Homeland Security, Science and Technology Directorate (DHS S&T) as part of the Remote Identity Validation Technology Demonstration (RIVTD).  For more information please visit [https://mdtf.org](https://mdtf.org) and [https://www.dhs.gov/science-and-technology/BI-TC](https://www.dhs.gov/science-and-technology/BI-TC)
+  version: 0.0.1
+  contact:
+    name: The MdTF
+    url: https://mdtf.org
+    email: info@mdtf.org
+  license:
+    name: Copyright (c) 2023, The Maryland Test Facility
+
+tags:
+  - name: Biometric Operations
+  - name: Algorithm Information
+
+components:
+
+  schemas:
+
+    Image:
+      type: object
+      description: |
+        Data transfer object for an image.
+      properties:
+        ImageData:
+          type: string
+          description: The captured image data in PNG or JPG format, encoded as a base64 string.
+          example: iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAIAAACQd1PeAAAAEElEQVR4nGJiYGAABAAA//8ADAADcZGLFwAAAABJRU5ErkJggg==
+          x-nullable: false
+      required:
+        - ImageData
+
+    Template:
+      type: object
+      description: |
+        Data transfer object for a template.
+      properties:
+        Template:
+          description: The template data, encoded as a base64 string.
+          type: string
```

Comment:
please add the line after this:
format: byte

Reply:
done